### PR TITLE
Login / logout commands 

### LINKED ISSUE #192: Improve publish resources to testnet
Developers should not have to sync the whole network to be able to publish resources to the testnet

- [x] add login and logout commands (https://github.com/mesg-foundation/js-sdk/pull/193)
- [ ] add `deploy:service` and `deploy:process`

### PR DESCRIPTION
**Pull request by antho1404**
Dependency: https://github.com/mesg-foundation/js-sdk/pull/188

Add command
- `login` that requires a mnemonic and a password and will save the credentials for a dedicated registry (for now localhost but should change to the testnet)
- `logout` that delete the credentials on the computer

### CODE REVIEW

File: packages/vault/CHANGELOG.md [hunk 1]
```diff
@@ -6,5 +6,9 @@
 #### Improvements
 
 - [#183](https://github.com/mesg-foundation/js-sdk/pull/183) Add localstorage store and add `keys` function to the store
+- [#](https://github.com/mesg-foundation/js-sdk/pull/) Add remove function to the vault
```
Comment (antho1404) [inline packages/vault/CHANGELOG.md]:
```suggestion
- [#193](https://github.com/mesg-foundation/js-sdk/pull/193) Add remove function to the vault
```

File: packages/vault/CHANGELOG.md [hunk 2]
```diff
@@ -6,5 +6,9 @@
...
+- [#](https://github.com/mesg-foundation/js-sdk/pull/) Add type for the vault's data
```
Comment (antho1404) [inline packages/vault/CHANGELOG.md]:
```suggestion
- [#193](https://github.com/mesg-foundation/js-sdk/pull/193) Add type for the vault's data
```

File: packages/vault/CHANGELOG.md [hunk 3]
```diff
@@ -6,5 +6,9 @@
...
 
 #### Bug fixes
+
+- [#](https://github.com/mesg-foundation/js-sdk/pull/) Automatically create the directory if needed for the file store
```
Comment (antho1404) [inline packages/vault/CHANGELOG.md]:
```suggestion
- [#193](https://github.com/mesg-foundation/js-sdk/pull/193) Automatically create the directory if needed for the file store
```

File: packages/cli/CHANGELOG.md [hunk 4]
```diff
@@ -3,15 +3,40 @@
...
-- [#183](https://github.com/mesg-foundation/js-sdk/pull/183) Add account commands `account:balance`, `account:create`, `account:export`, `account:list`
+- [#183](https://github.com/mesg-foundation/js-sdk/pull/183) Add account commands `account:balance`, `account:create`, `account:export`, `account:list` (reverted by #188)
 - [#181](https://github.com/mesg-foundation/js-sdk/pull/181) Add util to get hash from a transaction log
 - [#186](https://github.com/mesg-foundation/js-sdk/pull/186) Create/delete process based on LCD endpoint
-- [#187](https://github.com/mesg-foundation/js-sdk/pull/187) Add `account:transfer` command
-- [#187](https://github.com/mesg-foundation/js-sdk/pull/187) Transfer token when creating a process
+- [#187](https://github.com/mesg-foundation/js-sdk/pull/187) Add `account:transfer` command (reverted by #188)
+- [#187](https://github.com/mesg-foundation/js-sdk/pull/187) Transfer token when creating a process (reverted by #188)
+- [#](https://github.com/mesg-foundation/js-sdk/pull/) Add login/logout commands with synchronization in the user's configs
```
Comment (antho1404) [inline packages/cli/CHANGELOG.md]:
```suggestion
- [#193](https://github.com/mesg-foundation/js-sdk/pull/193) Add login/logout commands with synchronization in the user's configs
```

File: packages/cli/CHANGELOG.md [hunk 5]
```diff
@@ -29,13 +29,14 @@ Commands `service:dev` and `process:dev` are now smarter and automatically start
 
 #### Improvements
 
-- [#179](https://github.com/mesg-foundation/js-sdk/pull/179) Add new flag `lcd-port` to all command to configure the port of the LCD server.
+- [#179](https://github.com/mesg-foundation/js-sdk/pull/179) Add new flag `lcd-port` to all command to configure the port of the LCD server. (reverted by #188)
```
Comment (NicolasMahe) [inline packages/cli/CHANGELOG.md:32]: let's put link to the 188 PR
```suggestion
- [#179](https://github.com/mesg-foundation/js-sdk/pull/179) Add new flag `lcd-port` to all command to configure the port of the LCD server. (reverted by [#188](https://github.com/mesg-foundation/js-sdk/pull/188))
```

**Review by NicolasMahe — APPROVED** [review]

**Pull request merged**